web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.css

```diff
@@ -1,3 +1,7 @@
+.member-selector-dialog .custom-tenure-picker {
```

Comment:
This sets the width of a DatePicker that was added for selecting a custom tenure value.